src/logic/operate.js

```diff
@@ -0,0 +1,14 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '+': return Big(numberOne).add(numberTwo).c;
+    case '-': return Big(numberOne).minus(numberTwo).c;
+    case '*': return Big(numberOne).times(numberTwo).c;
+    case '/': return Big(numberOne).div(numberTwo).c;
+    case '%': return 100 * Big(numberOne).div(numberTwo).c;
```

Comment:
Job well done!

**1.** You need to wrap both the `numberOne` and `numberTwo ` with Big function, and also you can implement DRY code (that is; not repeating yourself.)

Example:
```
const operate = (num1, num2, operation) => {
  const num1 = Big(numberOne);
  const num2 = Big(numberTwo);
  switch (operation) {
    case '+': return (numberOne).add(numberTwo).c;
.........
```

**2.** `case '+': return (numberOne).add(numberTwo).c;` 
In the [documentation](https://mikemcl.github.io/big.js/#plus), you should use the word `plus` instead of `add`, if I may ask, can we also use `add`, you can share any doc that pointed to that, thanks.

Also, what is the `.c` workiing for? 
I'm curious you can also share doc on this too, thanks.

Reply:
Hi. I was getting the operation result from c, but I have seen in the documentation that It is better to use toFixed() because of the decimal separator. So I changed it.
Thanks for your help.